Gruntfile.js

```diff
@@ -13,7 +13,7 @@ module.exports = function(grunt) {
 
   var configs = require('load-grunt-configs')(grunt, {
     config: {
-      src: 'tasks/*.js',
+      src: 'tasks/configs/*.js',
```

Comment:
Good idea. The helper comment needs updating too: https://github.com/rpl/web-ext/blob/2e95f3dd3db4df77177c8a459965d61831860425/Gruntfile.js#L8